### PR TITLE
#167131517 Create static pages #167131517

### PR DESCRIPTION
**Pull request by Ochowo**
### What does this PR do?

* Create About us, how it works and Contact us Page

### Description of Task to be completed?

* Add component
* Write test for the component
* Create links from the homepage to each section on the page

### Any background context you want to provide?

N/A

### What are the relevant Github issues?

N/A

### Questions:

N/A

### Screenshot:

Screenshot 2019-07-16 at 11 57 00 AM
Screenshot 2019-07-16 at 11 57 08 AM
Screenshot 2019-07-16 at 11 57 16 AM

### CODE REVIEW

**Comment by beejay1293** [comment]
@Ochowo , just above the pictures of the trainers, you wrote `meet out trainers` instead of `meet our trainers`. kindly fix that, please

File: package.json [hunk 1]
```diff
@@ -23,6 +23,8 @@
   },
   "homepage": "https://github.com/andela/freyja-ah-frontend#readme",
   "dependencies": {
+    "@babel/polyfill": "^7.4.4",
+    "@babel/runtime": "^7.5.4",
```
Comment (beejay1293) [inline package.json:26]: @Ochowo we @babel/polyfill and @babel/runtime is redundant, we have handled that by adding @babel/plugin-transform-runtime to our .babelrc

File: src/views/AboutUs/aboutus.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    .handlesd,
+    .nam {
+      text-align: center;
+
+      .fci {
+        color: $headingcolor;
+      }
+
+      .fci:hover {
+        color: $bordercolor;
```
Comment (beejay1293) [inline src/views/AboutUs/aboutus.scss]: I believe there is a better way to implement this by using the power of sass
by doing this

```
.fci {
        color: $headingcolor;
      &:hover {
        color: $bordercolor;
      }
 }
```

**Review by beejay1293 — COMMENTED** [review]
nice job @ochowo, left some comments for you. 👍

File: package.json (hunk 1 above)
Comment (Ochowo) [reply package.json:26]: I already took them out but it came back after I rebased. I will take them out and push. thank you

**Review by fortunee — APPROVED** [review]
Thanks for the good work @Ochowo 👍

**Pull request merged**